app/app.go

```diff
@@ -361,18 +361,32 @@ func NewOsmosisApp(
 	// there is nothing left over in the validator fee pool, so as to keep the
 	// CanWithdrawInvariant invariant.
 	// NOTE: staking module is required if HistoricalEntries param > 0
+	// NOTE: capability module's beginblocker must come before any modules using capabilities (e.g. IBC)
```

Comment:
Was this just a bug before?

Reply:
Perhaps, idk.  I just copied this comment from simapp in cosmos-sdk